scanner.go

```diff
@@ -111,6 +111,7 @@ func scanWhitespace(b []byte) ([]byte, []byte) {
 	return b, b[len(b):]
 }
 
+//nolint:unparam
```

Comment:
scanComment - result 0 ([]byte) is never used (unparam)
Not sure if it's temporar or permanent, so I left it here. It basically says the first result is always ignored

Reply:
Not using it now, but we will when implementing the Document interface, which will allow to manipulate comments.